### PR TITLE
Book store5

### PR DESCRIPTION
**Pull request by arikarim**
The final milestone of Bookstore project:

- [x] style to navbar
- [x] style to each book
- [x] style to the form

### CODE REVIEW

File: src/components/categoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```
Comment (bellom) [inline src/components/categoryFilter.js]: Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

File: src/containers/ BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+BooksForm.propTypes = {
+  addBook: PropTypes.func.isRequired,
+};
+/*eslint-disable */
+const mapDispatchToProps = (dispatch) => (
+  bindActionCreators({
+    addBook: addBook,
+  }, dispatch)
+);
+/* eslint-enable */
```
Comment (bellom) [inline src/containers/ BooksForm.js]: Inline comment: 
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: README.md [hunk 3]
```diff
@@ -1,70 +1,55 @@
...
-## Available Scripts
+![image](./src/img/screen.png)
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+[Live Demo Link](https://book-storee.herokuapp.com/)
```
Comment (bellom) [inline README.md:10]: Seems you need to update your live demo link.
Push your updated code to Heroku.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @arikarim  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: README.md (hunk 3 above)
Comment (arikarim) [reply README.md:10]: Done.

File: src/index.js [hunk 4]
```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import './index.css';
-import App from './App';
+import App from './components/App';
+// import bookReducer from './reducers/book';
```
Comment (Forison) [inline src/index.js]: * [x] Let us remove commented code

File: src/components/categoryFilter.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    className="cates"
+    value={filter}
+    onChange={({ target: { value } }) => handleFilterChange(value)}
+  >
+    <option value="All">CATEGORIES</option>
+    { BOOKCATES.map((c) => <option key={c} value={c}>{ c }</option>) }
+  </select>
+);
+
+categoryFilter.propTypes = {
```
Comment (Forison) [inline src/components/categoryFilter.js:16]: * [x] Let us validate props, the block should not be empty

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Change Required❌

#👋 @arikarim, 

#### Live demo has been added to the readme, commented code was remove, and disabling comment for eslint have been removed. However there is something I think needs addressing before this project can be approved, Kindly check my comment below.

#### That is it, kindly make the changes and submit a re-review🤗.

#### Happy coding

Addo

File: src/components/categoryFilter.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+const categoryFilter = ({ filter, handleFilterChange }) => (
+  <select
+    className="cates"
+    value={filter}
+    onChange={({ target: { value } }) => handleFilterChange(value)}
+  >
+    <option value="All">CATEGORIES</option>
+    { BOOKCATES.map((c) => <option key={c} value={c}>{ c }</option>) }
+  </select>
+);
```
Comment (Forison) [inline src/components/categoryFilter.js:14]: * [ ] Let us add the prop validation for filter and handleFilterChange,
> I think the first is a string and the second a function

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change Required ❌ 

#### Good job on removing the commented code, however you completely removed the prop validation block, which should not be the case.

* [ ] Let make sure all props are validated, check my comment below.

#### That is it, kindly make the change and submit a re-review🤗

#### Happy coding

Addo Forison

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**